### ISSUE TITLE
HsvToRgb not working correctly

### ISSUE THREAD

**Issue by ChopsKingsland** [luisllamasbinaburo/Arduino-ColorConverter#5]
I am trying to convert the HSV colour (121,100,100) to RGB, and it gives the wrong RGB value... It outputs (156,172,172) whereas it should be (0,255,4)...

**Comment by berrak** [comment]
Hi there, I had a similar need to convert between HSV-RGB and RGB-HSV. Thus began writing an Arduino library and found your issue. In some sources its mentioned that h = [0,360], s = [0,1], v = [0,1]. Your example 'h' value is okay < 360, but I assumed 100i.e., end of the range. Thus I set both s=1.0 and v = 1.0. When I did the calculation with my to-be-released Arduino library, its results were 0.0, 255.0, and 4.25 after scaling back the normalized 's' and 'v' values by multiplying with 255 (end of the range). I will let you know when I put the library on Github. Cheers.
